extensions/azurePublishNew/src/node/azureResources/blobStorage.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { AZURE_HOSTING_GROUP_NAME } from '../getResources';
+import {
+  ProvisionMethod,
+  ProvisionWorkingSet,
+  ResourceDefinition,
+  ResourceProvisionService,
+} from '../provisionService';
+
+export const blobStorageDefinition: ResourceDefinition = {
+  key: 'blobStorage',
+  description:
+    'Azure blob storage provides scalable cloud storage, backup and recovery solutions for any data, including bot transcript logs.',
+  text: 'Azure Blob Storage',
+  tier: 'Standard_LRS',
```

Comment:
Are the tiers specific IDs or are they display strings?  If they are IDs then the UI can render the right display name and description, if they are names then I guess we should return something readable.  Consider having a set of constants for these to make sure they are consistent (just like the group name)

Reply:
these are display names. set them up in constants 👍